test/OpenSpaceToolkit/Physics/Time/Time.test.cpp

```diff
@@ -461,3 +449,17 @@ TEST(OpenSpaceToolkit_Physics_Time_Time, Parse)
         EXPECT_ANY_THROW(Time::Parse("-12:34:56", Time::Format::ISO8601));
     }
 }
+
+TEST(OpenSpaceToolkit_Physics_Time_Time, Seconds)
```

Comment:
Nit: didn't want to convert the decimal numbers you got from the unit tests above back into the hhmmss format you had originally specified above for simplicity?

Reply:
AI didn't want to do that lol.